src/api/spec/features/beta/webui/attributes_spec.rb

```diff
@@ -64,12 +64,14 @@
           login other_user
 
           visit index_attribs_path(project: user.home_project_name)
-          expect(page).not_to have_content('Add a new attribute')
+          expect(page).not_to have_content('Add Attribute')
         end
       end
 
       context 'with permissions' do
         scenario 'remove attribute' do
+          skip('Remove dialog is not working')
```

Comment:
I don't get it. Why skip it instead of fixing it?

Reply:
Or at least have a FIXME and an issue

Reply:
The fix was merged. I forgot to remove this. Thanks.

Reply:
Removed.